### ISSUE TITLE
How to run different simulator eg. iPhone 3.5-inch

### ISSUE THREAD

**Issue by samwize** [bendyworks/bwoken#70]
By default it runs on iPhone 4-inch.

**Comment by iToto** [comment]
Is there no way to specify which simulator to run on? Currently my project only builds on 32bit hardware. And even if I specify the Scheme to use, Bwoken always tries to build a 64bit version.

Is this related to this issue?

**Comment by ryanjm** [comment]
This has been solved with #79. 

If you run `instruments -s` it'll give you the list of simulators and devices it has access to. You can then use the name (or the device id) to specify it when you are running bwoken:

``` sh
$ bwoken test --simulator --family ipad --device "iPad Air (8.1 Simulator)"
```
